Jaron-Lane/goal-keeper
# Jl display habit data

# Description
Displaying habits to the Today's Habits page. Displaying a timer, and dummy data by means of HabitProvider and CompletedHabitProvider.

## Type of change

# Testing Instructions
git fetch --all
git checkout jl-display-habit-data
Should render habit cards with timers to the dashboard under the 'Habits to do Today' section, and should render CompHabit cards to 'Habits Completed Today' sections on the DOM. 

Timers should start and display a 'log' button once started. When the log buttons are pressed the timer should stop and the time value should be displayed in the console for future storing.

# Checklist:
